blt/src/Commands/HumsciCommand.php

```diff
@@ -241,4 +243,36 @@ protected function updateSite($multisite) {
     $this->say("Finished deploying updates to $multisite.");
   }
 
+  /**
+   * Update autoloader in composer.json.
+   *
+   * To allow us the ability to create TestBase.php files that can be inherited
+   * by test classes, we have to specify each namespace with the appropriate
+   * directory for the test files. Since composer doesn't do this dynamically,
+   * we have to manually build the autoloader data with all available tests.
+   *
+   * @command update-autoloader
+   */
+  public function updateAutoloader() {
+    $root = $this->getConfigValue('repo.root');
+    $humsci_modules = $this->getConfigValue('docroot') . '/modules/humsci';
+
+    $classes = [];
+    foreach ($this->rglob("$humsci_modules/*Test.php") as $path) {
+      $relative_path = str_replace("$root/", '', $path);
+
+      var_dump($relative_path);
```

Comment:
I'll pretend I didn't see this.

Reply:
![doh](https://media.giphy.com/media/TwtXMS5EnKDBK/giphy.gif)